### PR TITLE
fix(DB/creature): Adjust creature spawn location at Badlands

### LINKED ISSUE #5696: (DB/creature): Wrong NPC spawn location at Badlands
Original report at https://github.com/chromiecraft/chromiecraft/issues/562

#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [x] Alliance
- [x] Horde

##### CONTENT PHASE:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [x] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

Mobs spawned inside textures
DB GUID 6947
inside

DB GUID 7614 and DB GUID 6915
inside2

##### EXPECTED BLIZZLIKE BEHAVIOUR:

Mobs should not be spawned inside ground.

##### STEPS TO REPRODUCE THE PROBLEM:

1. Teleport to provided GUID
2. See spawn points for those mobs

##### EXTRA NOTES:

Type here.

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/6947789622de45479d2311fd5d1635af3a957b5d

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/98327454-db-creature-wrong-npc-spawn-location-at-badlands?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Si1ker**
## Changes Proposed:
-  Adjust XYZ pos

## Issues Addressed:

- Closes #5696
- Closes https://github.com/chromiecraft/chromiecraft/issues/562

## Tests Performed:

- Tested in-game

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Hacki95** [comment]
Where do you get those coordinates from? We should probably try to avoid changing spawns by hand without any type of source.

**Comment by Si1ker** [comment]
> Where do you get those coordinates from? We should try to avoid changing spawns by hand without any type of source.

Yes, by hand literally some meters where they were, this addressed the issue and is done in this way as having an NPC 5 meters at the right/left/up/below is not gamebreaking. Imagine how many NPCs are not with a blizzlike location in game.
No big deal n.n

**Comment by Hacki95** [comment]
Yes and it gets worse with every single spawn we do by hand. You can find the correct spawn in vmangos database for example, they have more accurate sniffed data.

One of the next projects I wanted to do was reimporting every herb/mining/chest node from different data because ours is so wrong. I don't want to do the same for creatures etc too.

**Comment by Si1ker** [comment]
> Yes and it gets worse with every single spawn we do by hand. You can find the correct spawn in vmangos database for example, they have more accurate sniffed data.
> 
> One of the next projects I wanted to do was reimporting every herb/mining/chest node from different data because ours is so wrong. I don't want to do the same for creatures etc too.

Again no big deal, the project will never be 100.0% accurate and as it was said by ac admins "even if a fix is not perfect but it improves the current behaviour, do it".
I don't have vmangos db, if you want/can contribute with the coordinates for those guids, feel free to 🐱

File: data/sql/updates/pending_db_world/rev_1620415939334068700.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1620415939334068700');
+
+UPDATE `creature` SET `position_x`=-7131.70, `position_y`=-3156.14, `position_z`=244.252 WHERE `guid`=6947;
+UPDATE `creature` SET `position_x`=-6663.84, `position_y`=-3469.57, `position_z`=255.870 WHERE `guid`=7614;
+UPDATE `creature` SET `position_x`=-6685.00, `position_y`=-3498.76, `position_z`=252.841 WHERE `guid`=6915;
```
Comment (Hacki95) [inline data/sql/updates/pending_db_world/rev_1620415939334068700.sql:5]: Use vmangos values or similiar

**Review by Hacki95 — CHANGES_REQUESTED** [review]

**Comment by Hacki95** [comment]
> 
> 
> > Yes and it gets worse with every single spawn we do by hand. You can find the correct spawn in vmangos database for example, they have more accurate sniffed data.
> > One of the next projects I wanted to do was reimporting every herb/mining/chest node from different data because ours is so wrong. I don't want to do the same for creatures etc too.
> 
> Again no big deal, the project will never be 100.0% accurate and as it was said by ac admins "even if a fix is not perfect but it improves the current behaviour, do it".
> I don't have vmangos db, if you want/can contribute with the coordinates for those guids, feel free to 🐱

I can check. But please don't submit any more hand changed coordinates without a source, I saw a few of them merged recently. Its true it will never be 100% accurate but if there is a way to get more accurate data we should definitely use it and not do it this way as it will make it just worse in the long run.

**Comment by Si1ker** [comment]
> > > Yes and it gets worse with every single spawn we do by hand. You can find the correct spawn in vmangos database for example, they have more accurate sniffed data.
> > > One of the next projects I wanted to do was reimporting every herb/mining/chest node from different data because ours is so wrong. I don't want to do the same for creatures etc too.
> > 
> > 
> > Again no big deal, the project will never be 100.0% accurate and as it was said by ac admins "even if a fix is not perfect but it improves the current behaviour, do it".
> > I don't have vmangos db, if you want/can contribute with the coordinates for those guids, feel free to 🐱
> 
> I can check. But please don't submit any more hand changed coordinates without a source, I saw a few of them merged recently. Its true it will never be 100% accurate but if there is a way to get more accurate data we should definitely use it as we could in this case. Hand changing thing should be a last resort.

> > > Yes and it gets worse with every single spawn we do by hand. You can find the correct spawn in vmangos database for example, they have more accurate sniffed data.
> > > One of the next projects I wanted to do was reimporting every herb/mining/chest node from different data because ours is so wrong. I don't want to do the same for creatures etc too.
> > 
> > 
> > Again no big deal, the project will never be 100.0% accurate and as it was said by ac admins "even if a fix is not perfect but it improves the current behaviour, do it".
> > I don't have vmangos db, if you want/can contribute with the coordinates for those guids, feel free to 🐱
> 
> I can check. But please don't submit any more hand changed coordinates without a source, I saw a few of them merged recently. Its true it will never be 100% accurate but if there is a way to get more accurate data we should definitely use it as we could in this case. Hand changing thing should be a last resort.

Sure, #5696 just assigned, will assign any incoming report of this nature, thanks for volunteering to make the project as blizzlike as possible! 🐱

**Comment by FrancescoBorzi** [comment]
good boys

**Pull request closed without merging**